src/openrct2/ride/TrackDesign.cpp

```diff
@@ -433,6 +433,8 @@ rct_string_id TrackDesign::CreateTrackDesignMaze(const Ride& ride)
 
     CoordsXY exitLoc = { location.x * 32, location.y * 32 };
     tileElement = map_get_first_element_at(location.x, location.y);
+    if (tileElement == nullptr)
+        return STR_NONE;
```

Comment:
`return STR_TRACK_TOO_LARGE_OR_TOO_MUCH_SCENERY;` Yeah I know it isn't the correct string but `STR_NONE` is used to indicate success in this function we need to indicate failure.